x/concentrated-liquidity/lp.go

```diff
@@ -77,6 +77,15 @@ func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 		}
 	}
 
+	hasFullPosition := k.hasFullPosition(ctx, poolId, owner, lowerTick, upperTick, joinTime, freezeDuration)
+
+	if !hasFullPosition {
+		err = k.initPositionUptime(ctx, poolId, owner, lowerTick, upperTick, liquidityDelta, joinTime, freezeDuration)
+		if err != nil {
+			return sdk.Int{}, sdk.Int{}, sdk.Dec{}, time.Time{}, err
+		}
+	}
```

Comment:
Why do we not error in the else case of this if it should not be possible to have the position with the same key due to `joinTime`?

Reply:
i didn't error for the <1min uptime case where one could modify position from 0sec to 60sec, but since we added 1nano second uptime, i'll add error